templates/slurm.conf.erb

```diff
@@ -228,8 +228,9 @@ Licenses=<%= scope['slurm::licenses'] %>
 
 
 
-#MaxJobCount=5000
 #MaxStepCount=40000
+MaxArraySize=<%= scope['slurm::maxarraysize'] %>
+MaxJobCount=<%= scope['slurm::maxjobcount'] %>
```

Comment:
As above; 

I would suggest to leave the same approach as for the other parameters: if the value is the default one, leave it as comment. In this case: 

```ruby
<% if scope['slurm::maxjobcount'] == scope['slurm::params::maxjobcount']  -%>
# MaxJobCount=
<% else -%>
MaxJobCount=<%= scope['slurm::maxjobcount'] %>
<% end -%>
```

Reply:
As above, `<% if scope['slurm::maxjobcount'] -%>` ?